fast_api_als/routers/lead_conversion.py

```diff
@@ -47,15 +49,17 @@ async def submit(file: Request, token: str = Depends(get_token)):
 
     if 'lead_uuid' not in body or 'converted' not in body:
         # throw proper HTTPException
-        pass
+        logging.error("'lead_uuid' or 'converted' not found")
+        raise HTTPException(status_code=500, detail="'lead_uuid' or 'converted' not found")
```

Comment:
Previously I thought that, as these statements
```    
    lead_uuid = body['lead_uuid']
    converted = body['converted']
```
are using that data which wasn't available so it can also be thought of as a situation where the server doesn't know how to handle it. I also thought of 400 as the other possible option based on the description of response_code 400